CONTRIBUTING.md

```diff
@@ -41,8 +30,8 @@ and clone the fork which will make opening PRs easier.
 
 ### Using Xcode (easiest)
 
-1. Install the latest beta version of
-   [Xcode](https://developer.apple.com/xcode).
+1. Install Xcode 16 or newer. You can find Xcode on the [Apple Developer](https://developer.apple.com/xcode/)
```

Comment:
```suggestion
1. Install Xcode 16 or newer, from the [Apple Developer](https://developer.apple.com/xcode/)
```

Not a very strong suggestion, I just think it's a bit more straightforward.

Reply:
Sure, will tweak.